src/assertions/elContainsText.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+import util from "util";
+import clc from "cli-color";
+
+import selectorUtil from "../util/selector";
+import BaseAssertion from "../base-assertion";
+
+let ElContainsText = function (nightwatch = null, customized_settings = null) {
```

Comment:
all commands and assertions cannot be es6 class. nightwatchjs wont read it as all es6 class methods are not enumerable.